emails/migrations/0058_profile_onboarding_free_state.py

```diff
@@ -0,0 +1,161 @@
+# Generated by Django 3.2.19 on 2023-10-19 23:46
+
+from django.db import migrations, models
+
+
+def add_db_default_forward_func(apps, schema_editor):
+    """
+    Add a database default of false for sent_welcome_email, for PostgreSQL and SQLite3
+    Note: set sent_welcome_email = true for existing users
+
+    Using `./manage.py sqlmigrate` for the SQL, and the technique from:
+    https://stackoverflow.com/a/45232678/10612
+    """
+    if schema_editor.connection.vendor.startswith("postgres"):
+        schema_editor.execute(
+            'ALTER TABLE "emails_profile"'
+            ' ALTER COLUMN "onboarding_free_state" SET DEFAULT 0;'
+        )
+        # Set all existing profiles to 0 so we don't send them another welcome email
```

Comment:
```suggestion
        # Set all existing profiles to 0 so existing users also see the free onboarding
```
Seems like copy paste error. Not sure if this will end up being the case since Product and Design is reconsidering who should see the free onboarding but is nice to have context on previous decisions.